### PR TITLE
fizzbuzz complete

### PR DESCRIPTION
**Pull request by lostybtw**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

Merge This Pwease?

### CODE REVIEW

**Comment by lostybtw** [comment]
@Kush1101

**Review by CaedenPH — APPROVED** [review]

File: dynamic_programming/fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    """
+
+    if not type(iterations) == int:
+        raise ValueError("iterations must be defined as integers")
+    if not iterations >= 1:
+        raise ValueError(
+            """iterations must be done more
+                         than 0 times to play FizzBuzz"""
+        )
```
Comment (cclauss) [inline dynamic_programming/fizz_buzz.py:37]:
```suggestion
        raise ValueError(
            "Iterations must be done more than 0 times to play FizzBuzz"
        )
```

File: dynamic_programming/fizz_buzz.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def fizz_buzz(number: int, iterations: int) -> str:
+    """
+    Plays FizzBuzz.
+    Prints Fizz if number is a multiple of 3.
+    Prints Buzz if its a multiple of 5.
+    Prints FizzBuzz if its a multiple of both 3 and 5 or 15.
+    Else Prints The Number Itself.
+
+    >>> fizz_buzz(1,7)
+    '1 2 Fizz 4 Buzz Fizz 7 '
```
Comment (cclauss) [inline dynamic_programming/fizz_buzz.py:13]: We need a test that generates `FizzBuzz`.

We need a test with negative numbers.

We need a test with floating point numbers.

Comment (lostybtw) [reply dynamic_programming/fizz_buzz.py:13]: Okay, I'll be on it/

**Comment by lostybtw** [comment]
@cclauss Can You Review?

**Review by cclauss — APPROVED** [review]
